wallet-webapps/src/main/webapp/vue-app/components/wallet-app/TransactionHistoryChartSummary.vue

```diff
@@ -78,12 +81,33 @@ export default {
   data() {
     return {
       selectDateMenu: false,
+      id: `DatePicker${parseInt(Math.random() * 10000)
+          .toString()
+          .toString()}`,
```

Comment:
```suggestion
      id: `DatePicker${parseInt(Math.random() * 10000)}`,
```

Reply:
you are right we don't need to add `toString()`